### ISSUE TITLE
More disambiguiation support in the search bar

### ISSUE THREAD

**Issue by mgaudet** [Pernosco/pernosco#50]
In Matrix, khuey was saying you can call functions from the search bar, which blew my mind. 

However, in the [trace where I tested](https://pernos.co/debug/Ed7xfNyCYI_cKaGFO-BosQ/index.html#f%7Bm%5BBFI8,ASIS+Q_,t%5BAWc,AjyA_,f%7Be%5BBFI8,ASHf4Q_,s%7Baf0MUuKAA,bMCA,uZWtM,obYPQ___/) I was unable to get the search bar to invoke [`js::DumpValue`](https://searchfox.org/mozilla-central/source/js/src/util/DumpFunctions.cpp#195); I tried entering `js::DumpValue(init.mReason)`, and got the error `failed to evaluate: Ambiguious identifier`

**Comment by khuey** [comment]
Yeah it looks like there's a few different overloads of `js::DumpValue` and we don't currently have support for picking amongst them.
